scripts/website/shared/shared.go

```diff
@@ -0,0 +1,16 @@
+package shared
```

Comment:
Could we consider a more descriptive package name? According to the Go blog on [package names](https://go.dev/blog/package-names), `shared` might be perceived as somewhat generic.